### ISSUE TITLE
Add like , comment and save post -- these stuffs to the feed . It'd be great .

### ISSUE THREAD

**Issue by nsrs-78** [Mayuresh-22/Minglr#49]

**Comment by nsrs-78** [comment]
Can I contribute? Let me know then :)

**Comment by Thanush19** [comment]
i can wok on this with adding more features

**Comment by nsrs-78** [comment]
Ok then I'll be working with Share and save post options.

**Comment by Mayuresh-22** [comment]
@nsrs-78 yes you can start working with it. I'll assign this to you.

**Comment by nsrs-78** [comment]
Tnxx

**Comment by Mayuresh-22** [comment]
@nsrs-78 are you still working on the issue?

**Comment by nsrs-78** [comment]
not started yet

**Comment by Parul1606** [comment]
Hello @Mayuresh-22 can you please assign me this issue i would like to work on it.

**Comment by Mayuresh-22** [comment]
@Thanush19 are you willing to work on this issue?

**Comment by Mayuresh-22** [comment]
Hello @Parul1606, @GaganMeshram123, @Thanush19 if you want to work on this issue reply this comment and I'll assign on the basis of FCFS

**Comment by GaganMeshram123** [comment]
> Hello @Parul1606, @GaganMeshram123, @Thanush19 if you want to work on this issue reply this comment and I'll assign on the basis of FCFS

Yes @Mayuresh-22  I want to work on these issue

**Comment by GaganMeshram123** [comment]
@Mayuresh-22 I can work on these issue

**Comment by Mayuresh-22** [comment]
okay @GaganMeshram123, I will assign this issue to you right away.

**Comment by GaganMeshram123** [comment]
@Mayuresh-22 yes

**Comment by GaganMeshram123** [comment]
@Mayuresh-22  can you guide me for these issue

**Comment by Mayuresh-22** [comment]
Yes just start working and I'll try my best to assist you with it.

**Comment by Mayuresh-22** [comment]
@GaganMeshram123 any updates?

**Comment by GaganMeshram123** [comment]
Yes merging pull request
